recipes/ligo-remote-epics-scripts/meta.yaml

```
@@ -0,0 +1,35 @@
+{% set name = "ligo-remote-epics-scripts" %}
+{% set version = "0_0_3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version|replace('_','.') }}
+
+source:
+  url: https://git.ligo.org/cds/ligo-remote-epics-scripts/-/archive/v0.0.3/ligo-remote-epics-scripts-v0.0.3.tar.gz
+  sha256: 6a40f38c1c6ccdeee646caf1ca34399d1a302db942e7cfcb110fa79f6631556e
+
+build:
+  number: 0
+  skip: true  # [win]
+
```

The scrips invoke `medm`, so that at least needs to be a runtime requirement

```suggestion

requirements:
  run:
    - ligo-medm

```